js/app.js

```diff
@@ -171,7 +171,7 @@ function createSecondaryWeightsForm(totalNumber) {
     saveUpdatedUserInfo(getGlobalUsername(), userData);
     show(lookupUser(getGlobalUsername()));
   });
-
+  
```

Comment:
Your lintier should have had a problem with this extra space. Are you checking the errors/warnings before you push?

Reply:
(could you remove it)